### PR TITLE
Add EE10 repeats for non Jakarta features phase 1

### PR DESCRIPTION
**Pull request by jhanders34**
Add repeats for fats that test the following features:
- acmeCA-2.0
- adminCenter-1.0
- audit-1.0
- batchManagement-1.0
- constrainedDelegation-1.0
- grpc-1.0 / grpcClient-1.0
- passwordUtilities-1.0 and 1.1
- restHandler-1.0
- sessionDatabase-1.0

For: #18722

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

**Review by jvanhill — DISMISSED** [review]
The repeats for the security components look good.

**Comment by amyreit** [comment]
The ui changes look good to me.

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/JACCFatUtils.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import java.nio.file.Path;
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/JACCFatUtils.java:15]: Copyright date range update?

File: dev/com.ibm.ws.security.jaspic_fat/build.gradle [hunk 2]
```diff
@@ -263,9 +263,15 @@ autoFVT.doLast {
      into new File(autoFvtDir, 'publish/bundles')
```
Comment (pnicolucci) [inline dev/com.ibm.ws.security.jaspic_fat/build.gradle:263]: Copyright date range update?

File: dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/JACCFatUtils.java (hunk 1 above)
Comment (jhanders34) [reply dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/JACCFatUtils.java:15]: Done

File: dev/com.ibm.ws.security.jaspic_fat/build.gradle (hunk 2 above)
Comment (jhanders34) [reply dev/com.ibm.ws.security.jaspic_fat/build.gradle:263]: Done

**Review by pnicolucci — DISMISSED** [review]

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby #build

**Review by Channyboy — DISMISSED** [review]

**Review by cgianfra — DISMISSED** [review]

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#libby #build

**Comment by jhanders34** [comment]
#libby

**Review by jvanhill — APPROVED** [review]

**Pull request merged**